### PR TITLE
feat(plugin-server): allow historical exports to be enabled and disabled via env

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Problem

When we've had to disable this in Cloud, we just got rid of the function call, but that can then affect self-hosted as well.

This should just be an env var.

## Changes

Enable historical exports if var is set

## How did you test this code?

Tests exist for historical exports

### CODE REVIEW

File: task-definition.plugins.json [hunk 1]
```diff
@@ -99,6 +99,10 @@
                 {
                     "name": "KAFKA_PARTITIONS_CONSUMED_CONCURRENTLY",
                     "value": "8"
+                },
+                {
+                    "name": "HISTORICAL_EXPORTS_ENABLED",
+                    "value": "False"
```
Comment (tiina303) [inline task-definition.plugins.json:105]: shouldn't this be True atm?

**Review by tiina303 — COMMENTED** [review]
n00b q: the VM is created on restart, and to change a flag we'd need to push a PR too, so this works, but I'm wondering if it would be better to gate within the historical exports function? i.e. plugins call some function that is essentially (which could allow us to change the flag on the instance if we wanted to):
```
def plugin_historical_export(params):
  if not HISTORICAL_EXPORT_ENABLED:
    throw "Historical Export disabled by flag"
  actual_historical_export_function(params)
```

File: task-definition.plugins.json (hunk 1 above)
Comment (yakkomajuri) [reply task-definition.plugins.json:105]: Nobody is complaining and we haven't had a chance to go deeply into it so while I'm reasonably sure it works, I'd like to keep it off for now

**Comment by yakkomajuri** [comment]
Hmm I disagree with you there because else users will see a button and want to click it. I rather they not see a button at all, and be unaware that this feature exists (unless ofc they've seen it in the past)

File: task-definition.plugins.json (hunk 1 above)
Comment (tiina303) [reply task-definition.plugins.json:105]: ok, I didn't know it was off atm sounds good

**Review by tiina303 — APPROVED** [review]

**Pull request merged**